.gitignore

```diff
@@ -5,3 +5,9 @@ coredns
 kubectl
 go-test-tmpfile*
 coverage.txt
+.idea/
+.vscode/
+cover.out
+cover.txt
+coverage.out
+myhosts
```

Comment:
what's myhost?

Reply:
I didn't intend to commit .gitignore. reverting.